From 70f656d526f9753df8e2d949e64d075a5a245334 Mon Sep 17 00:00:00 2001
From: Andrew kim <andrew.kim@intel.com>
Date: Wed, 16 Jan 2019 12:06:15 -0800
Subject: [PATCH 2287/2328] net: sched: fix build issue on sch_taprio file

fix the incompatible pointer type from taprio_graft()
and taprio_init()

Signed-off-by: Andrew kim <andrew.kim@intel.com>
---
 net/sched/sch_taprio.c | 8 +++-----
 1 file changed, 3 insertions(+), 5 deletions(-)

diff --git a/net/sched/sch_taprio.c b/net/sched/sch_taprio.c
index 51b47dd..22edfd6 100644
--- a/net/sched/sch_taprio.c
+++ b/net/sched/sch_taprio.c
@@ -698,8 +698,7 @@ static void taprio_destroy(struct Qdisc *sch)
 	}
 }
 
-static int taprio_init(struct Qdisc *sch, struct nlattr *opt,
-		       struct netlink_ext_ack *extack)
+static int taprio_init(struct Qdisc *sch, struct nlattr *opt)
 {
 	struct taprio_sched *q = qdisc_priv(sch);
 	struct net_device *dev = qdisc_dev(sch);
@@ -734,7 +733,7 @@ static int taprio_init(struct Qdisc *sch, struct nlattr *opt,
 	if (!opt)
 		return -EINVAL;
 
-	return taprio_change(sch, opt, extack);
+	return taprio_change(sch, opt, NULL);
 }
 
 static struct netdev_queue *taprio_queue_get(struct Qdisc *sch,
@@ -750,8 +749,7 @@ static struct netdev_queue *taprio_queue_get(struct Qdisc *sch,
 }
 
 static int taprio_graft(struct Qdisc *sch, unsigned long cl,
-			struct Qdisc *new, struct Qdisc **old,
-			struct netlink_ext_ack *extack)
+			struct Qdisc *new, struct Qdisc **old)
 {
 	struct taprio_sched *q = qdisc_priv(sch);
 	struct net_device *dev = qdisc_dev(sch);
-- 
2.7.4

